roles/docker/defaults/main.yml

```diff
@@ -6,3 +6,4 @@ docker_version: 1.9.1
 docker_rule_comment: "docker api"
 docker_device: ""
 docker_device_size: "10000MB"
+docker_device_size: "1000MB"
```

Comment:
@vvb did you mean to define `docker_device_metadata_size: "1000MB"` here? Otherwise looks like `docker_device_size` is getting defined twice.